packages/playground/src/dashboard/components/user_farms.vue

```diff
@@ -99,12 +99,14 @@
                   <input-validator
                     :value="address"
                     :rules="[validators.required('Address is required.'), customStellarValidation]"
+                    :async-rules="[() => validators.isValidStellarAddress(address)]"
```

Comment:
```suggestion
                    :async-rules="[validators.isValidStellarAddress]"
```

we should be able to pass a ref to function directly